CHANGELOG.md

```diff
@@ -37,6 +37,7 @@
   * Move system state out of running software directory
   * Fix updates on systems with an API password set
   * Fix the display service on systems with an API password set
+  * Add toggleable option to persist logs
```

Comment:
This might need to move to a different section now that it's no longer in the webapp

Reply:
Especially because this is now getting into a release a version or two later 🥴